### PR TITLE
begin work on discord.py@v2

### PR DESCRIPTION
**Pull request by bijij**
### Description
Adds support for discord.py 2.0

### Checklist

- [x] This PR has been tested.

### CODE REVIEW

**Comment by bijij** [comment]
Given DPY 2.0 currently does not have a semi-stable api, might be worth conscidering making a v2.0 branch?

File: discord/ext/alternatives/binary_checks.py [hunk 1]
```diff
@@ -26,8 +26,6 @@ async def no(ctx):
 ```
 """
 
-import inspect
```
Comment (ShineyDev) [inline discord/ext/alternatives/binary_checks.py:29]: I think these changes should go elsewhere.

File: discord/ext/alternatives/dict_converter.py [hunk 2]
```diff
@@ -20,7 +20,6 @@ async def ban(ctx, **users_reasons_mapping: typing.Dict[discord.Member, str]):
 ```
 """
 
-import inspect
```
Comment (ShineyDev) [inline discord/ext/alternatives/dict_converter.py:23]: Here too.

File: discord/ext/alternatives/messageable_wait_for.py [hunk 3]
```diff
@@ -1,5 +1,3 @@
-from discord.abc import Messageable
-from discord.message import Message
```
Comment (ShineyDev) [inline discord/ext/alternatives/messageable_wait_for.py:2]: And here.

File: discord/ext/alternatives/object_contains.py [hunk 4]
```diff
@@ -36,7 +36,6 @@
 
 from discord.guild import Guild
 from discord.member import Member
-from discord.message import Message
```
Comment (ShineyDev) [inline discord/ext/alternatives/object_contains.py:39]: And here.

**Review by ShineyDev — CHANGES_REQUESTED** [review]

**Review by ShineyDev — APPROVED** [review]
LGTM.

**Pull request merged**